source/object-model/computed-properties-and-aggregate-data.md

```diff
@@ -78,12 +77,11 @@ export default Ember.Controller.extend({
 });
 ```
 
-Note here that we use the special key `[]` instead of `@each.foo` (which is used for observing 
-a specific dependent key, such as `foo`, on each item). This instructs Ember.js to only care about 
-the array itself changing (such as adding/removing items), rather than changes to the items themselves.
+Here, `indexOfSelectedTodo` depends on `todos.[]`, so will update if we add an item
```

Comment:
i think `when` here creates expectation that something must happen, whereas it may not, so i feel `if` is a little more neutral and appropriate.  but a very subtle diff, so if you have strong feeling for `when` we can change.